tokio-trace/src/macros.rs

```diff
@@ -2,19 +2,23 @@
 ///
 /// # Examples
 ///
-/// Creating a new span with no fields:
+/// Creating a new span:
 /// ```
-/// # #[macro_use]
-/// # extern crate tokio_trace;
-/// # use tokio_trace::Level;
+/// #[macro_use]
+/// extern crate tokio_trace;
```

Comment:
Should this use 2018 syntax?

Reply:
We still have a CI build against Rust 1.26.0, so I don't think we can use the 2018 edition yet?